examples/README.rst

```diff
@@ -1,4 +1,4 @@
-.. image:: https://www.giotto.ai/static/vector/logo.svg
+.. image:: https://www.giotto.ai/static/vector/logo-tda.svg
```

Comment:
Could we use the local version of the logo?

Reply:
Why not!